lib/model/model.go

```diff
@@ -2613,6 +2615,13 @@ func (m *model) checkDeviceFolderConnectedLocked(device protocol.DeviceID, folde
 	return nil
 }
 
+func (m *model) saveConfig() {
+	if err := m.cfg.Save(); err != nil {
```

Comment:
However bad this sounds, my gut says we should make .Save() require a logger, or NewWrapper require a logger.
It's way too easy to just m.cfg.Save().

I think the config is boot strapped high enought that we should be able to inject the logger into the wrapper.